src/proto/move_robot_and_ball_messages.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package TbotsProto;
+
+import "proto/geometry.proto";
+
+message MoveRobotMessage
+{
+    Point global_position                   = 1;
+    Angle global_orientation                = 2;
+    Vector global_velocity                  = 3;
+    AngularVelocity global_angular_velocity = 4;
```

Comment:
lets add another field here for Team color, looks like that's also needed to figured out which team to control

```
enum TeamColour                                                                                                                                                                                           
{                                                                                                                                                                                                                                                                                                                                                                          
    YELLOW = 1;                                                                                                                                                                                     
    BLUE = 2;                                                                                                                                                                                       
}   
```

And then add a field

`TeamColour team_colour = 5`;